Description: disable diesel feature due to error
 Enabling it results in:
 .
 error[E0212]: cannot use the associated type of a trait with uninferred generic parameters
   --> src/features/diesel.rs:27:28
    |
 27 |     fn from_sql(bytes: DB::RawValue<'_>) -> deserialize::Result<Self> {
    |                            ^^^^^^^^^^^^^^^^
    |
 help: use a fully qualified path with inferred lifetimes
    |
 27 |     fn from_sql<'v>(bytes: <DB as backend::private::HasRawValue<'_>>::RawValue<'v>) -> deserialize::Result<Self> {
    |                            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 .
 Testing with latest diesel is good so presumably this could be dropped after
 its update.
Forwarded: not-needed
Last-Update: 2024-08-31
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -63,6 +63,6 @@
 
-[dependencies.diesel]
-version = "2"
-optional = true
-default-features = false
+#[dependencies.diesel]
+#version = "2"
+#optional = true
+#default-features = false
 
@@ -165,3 +165,3 @@
 default = ["std"]
-diesel = ["dep:diesel"]
+#diesel = ["dep:diesel"]
 markup = ["dep:markup"]
